### ISSUE TITLE
[@types/newman] postman-collection has no exported member CollectionDefinition, VariableScopeDefinition

### ISSUE THREAD

**Issue by doradsoft** [DefinitelyTyped/DefinitelyTyped#48543]
- [x] I tried using the `@types/newman` package and had problems.
- [x] [Mention](https://github.com/blog/821-mention-somebody-they-re-notified) the authors (see `Definitions by:` in `index.d.ts`) so they can respond.
  - Authors: @LogvinovLeon, @Graham42

```
node_modules/@types/newman/index.d.ts:12:5 - error TS2305: Module '"postman-collection"' has no exported member 'CollectionDefinition'.
12     CollectionDefinition,
       ~~~~~~~~~~~~~~~~~~~~
node_modules/@types/newman/index.d.ts:14:5 - error TS2305: Module '"postman-collection"' has no exported member 'VariableScopeDefinition'.
14     VariableScopeDefinition
       ~~~~~~~~~~~~~~~~~~~~
```

**Comment by cabelitos** [comment]
any news on this one?

**Comment by rohit-gohri** [comment]
This started happening because postman-collection started publishing their own official types which are incompatible with existing @types definition: https://github.com/postmanlabs/postman-collection/pull/1076

**Comment by rohit-gohri** [comment]
Downgrade postman-collection to fix:

`npm i --save-dev postman-collection@3.6.4`

**Comment by rohit-gohri** [comment]
Should be fixed in postman-collection@3.6.9

https://github.com/postmanlabs/postman-collection/issues/602#issuecomment-753507973

**Comment by orta** [comment]
Hi thread, we're moving DefinitelyTyped to use [GitHub Discussions](https://github.com/DefinitelyTyped/DefinitelyTyped/issues/53377) for conversations the `@types` modules in DefinitelyTyped.

To help with the transition, we're closing all issues which haven't had activity in the last 6 months, which includes this issue. If you think closing this issue is a mistake, please pop into the [TypeScript Community Discord](https://discord.gg/typescript) and mention the issue in the `definitely-typed` channel.
